service/cfg_test.go

```diff
@@ -0,0 +1,19 @@
+package main
+
+import (
+	"testing"
+	"time"
+)
+
+func TestCfgParse(t *testing.T) {
```

Comment:
> It's better to use `ioutil.TempFile` to write test files. See https://github.com/tarantool/cartridge-cli/blob/master/cli/project/dockerfiles_test.go for example.
> 
> It allows to remove `test_cfg` files and others.

Thanks for the hint. I deleted the `test_cfg.json` file, but I prefer to keep "test_instances" because:
- It is more convenient to work with such an "instance file" and it is convenient to use it for manual integration testing when developing functionality.
- I plan to reuse this file in integration tests.

Reply:
Updated.